config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -10,9 +10,61 @@ periodics:
   - org: kubernetes-sigs
     repo: cluster-api-provider-aws
     base_ref: master
-    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
+    path_alias: sigs.k8s.io/cluster-api-provider-aws
```

Comment:
Why this change?

Reply:
not intentional but seems like it doesn't matter.